internal/controller/reconciler_test.go

```diff
@@ -232,9 +231,8 @@ var _ = Describe("Reconciler", func() {
 				Consistently(eventCh).ShouldNot(Receive())
 				Expect(resultCh).To(Receive(Equal(result{err: nil, reconcileResult: reconcile.Result{}})))
 			},
-			Entry("Upserting valid HTTPRoute", getReturnsHRForHR(hr1), 0, hr1NsName),
```

Comment:
This change should have happened in https://github.com/nginxinc/nginx-kubernetes-gateway/commit/d121ff136620844826ae5d20024e876af6c99119 which undid https://github.com/nginxinc/nginx-kubernetes-gateway/commit/e36064525e6f00b3582c5de099bffe4b4a6db833